### PR TITLE
Update versions for 7.0.0 RTM

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

**Comment by Pilchie** [comment]
Are we doing this for the full stack already?  If so, then fine, but it's a bit of a pain since it breaks the ability to install daily builds.

**Comment by dougbu** [comment]
> Are we doing this for the full stack already? If so, then fine, but it's a bit of a pain since it breaks the ability to install daily builds.

Ugh, I got ahead of myself. The versions are changing but not being stabilized yet.

File: eng/Versions.props [hunk 1]
```diff
@@ -9,15 +9,14 @@
...
     <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
-    <ValidateBaseline>true</ValidateBaseline>
+    <ValidateBaseline>false</ValidateBaseline>
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```
Comment (dougbu) [inline eng/Versions.props:16]:
```suggestion
    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

File: eng/Versions.props [hunk 2]
```diff
@@ -9,15 +9,14 @@
     <AspNetCoreMajorVersion>7</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
-    <ValidateBaseline>true</ValidateBaseline>
+    <ValidateBaseline>false</ValidateBaseline>
```
Comment (dougbu) [inline eng/Versions.props:12]: Don't need this change either
```suggestion
    <ValidateBaseline>true</ValidateBaseline>
```

**Pull request merged**